scripts/verify

```diff
@@ -55,6 +55,75 @@ done
 [[ -z "$parameters" ]] && parameters="{}"
 [[ -z "$wait_timeout" ]] && wait_timeout=600
 
+function delete_namespace() {
+  if [[ -z "$namespace_deleted" ]]; then
+    echo "INFO Collecting events for namespace \"$NAMESPACE\""
+    kubectl get events --namespace="$NAMESPACE" || echo "ERROR Failed to get events for namespace \"$NAMESPACE\""
+    error_events="$(kubectl get events --namespace="$NAMESPACE" --field-selector="type!=Normal" || echo "ERROR Failed to get events for namespace \"$NAMESPACE\"")"
+
+    if [[ ! -z $deployer_name ]]; then
+      echo "INFO Collecting logs for deployer"
+      deployer_logs="$(kubectl logs "jobs/$deployer_name" --namespace="$NAMESPACE" || echo "ERROR Failed to get logs for deployer $deployer_name")"
+      echo "$deployer_logs"
+
+      deployer_errors="$(echo "$deployer_logs" | /scripts/filter_deployer_logs.py)"
+    fi
+
+    echo "INFO Deleting namespace \"$NAMESPACE\""
+    kubectl get all --namespace "$NAMESPACE" --output=yaml
+    kubectl delete namespace "$NAMESPACE"
+    namespace_deleted="true"
+  fi
+}
+
+function clean_resources() {
+  if [[ -z "$resources_cleaned" ]]; then
+    echo "INFO cleaning all resources."
+    delete_namespace
+    if [[ ! -z "$deployer_role_name" ]]; then
+      kubectl delete clusterrolebinding "$deployer_role_name"
+    fi
+    resources_cleaned="true"
+  fi
+}
+
+function clean_and_exit() {
+  if [[ -n "$1" ]]; then
+    echo -e "$1"
+    error_summary="$error_summary$1\n"
+  fi
+  clean_resources
+  exit 1
+}
+
+function print_summary() {
+  if [[ -n "$error_events" ]]; then
+    error_summary="${error_summary}Error events found in namespace \"$NAMESPACE\"\n"
```

Comment:
It's safer to use `printf` directly here to turn `\n` into new lines, rather than `echo -e` on the entire log